### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by oerkperl**
The home chart for bitcoin.

### CODE REVIEW

File: app/components/home/charts/Charts.tsx [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const Charts: React.FC = () => {
           `https://api.coingecko.com/api/v3/coins/bitcoin/market_chart?${params}`
         );
         setPriceData(data.prices);
+        console.log(data.prices);
```
Comment (yoloOnTheBattlefield) [inline app/components/home/charts/Charts.tsx]: remove

File: app/components/styled.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export const Main = styled.main`
+max-width:1200px;
+margin:0 auto;
+padding:0 .5rem;
+
+`
+export const Section = styled.section<{$margin?:string|null}>`
+width:100%;
+margin:${props => props.$margin};
+padding:0 .5rem;
```
Comment (yoloOnTheBattlefield) [inline app/components/styled.ts]: indent

File: app/components/styled.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+export const Col = styled.div<{$width?:string}>`
+  box-sizing: border-box;
+  min-height: 20px;
+  width: ${(props) => props.$width};
+  float: left;
+`;
+
+export const HorizontalLine = styled.hr`
+color:#888
```
Comment (yoloOnTheBattlefield) [inline app/components/styled.ts]: indent

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]
https://www.loom.com/share/d22c5fafab3440768f0313a46ee2e187?from_recorder=1&focus_title=1

File: app/components/styled.ts (hunk 3 above)
Comment (oerkperl) [reply app/components/styled.ts]: Addressed.

**Pull request merged**